_features/lin_advance.md

```diff
@@ -133,12 +172,31 @@ The shown G-code Start Scripts are individual to each printer and personal taste
 <br><br>
 ## Developer Information
 
+### General Informations
+
 The force required to push the filament through the nozzle aperture depends, in part, on the speed at which the material is being pushed into the nozzle. If the material is pushed faster (=printing fast), the filament needs to be compressed more before the pressure inside the nozzle is high enough to start extruding the material.
 
 For a single, fast printed line, this results in under-extrusion at the line's starting point (the filament gets compressed, but the pressure isn't high enough) and over-extrusion with a blob at the end of the line (the filament is still compressed when the E motor stops, resulting in oozing).
 
 For a complete print, this leads to bleeding edges at corners (corners are stop/end points of lines) and in extreme cases even gaps between perimeters due to under-extrusion at their starting points.
 
+### Version 1.5
+Version 1.5 handles the pressure correction slightly different to reach the following goals:
+- respect extruder jerk
+- ensure a smooth extruder movement without rattling
+- reduce load inside stepper ISR
+
+It does this by calculating an extruder speed offset within the planner for the given segment. If we have a true, linear acceleration then this will execute the needed advance steps just in time, so we have reached all our needed advance steps just when the acceleration part is over. As Marlin uses an approximation for the acceleration calculation inside the ISR, this is not perfectly true, we will came to this later.
+If the needed speed offset would exceed the allowed extruder jerk, the print acceleration for this segment is reduced to a lower value so extruder jerk is not exceeded any more. This is compareable to the check Marlin does for each axis, for example if we have a print acceleration of 2000mm/s² but X axis max. acceleration is set to 500mm/s², print acceleration is reduced to 500mm/s². During the trapezoid calculation, `LIN_ADVANCE` calculates the needed amount of advance steps at cruising speed and at final speed (=speed at and of the block).
```

Comment:
"This is **compareable** to the check" --> "This is **comparable** to the check"